nx-X11/programs/Xserver/include/dix.h

```diff
@@ -90,47 +90,32 @@ SOFTWARE.
 
 /* XXX if you are using this macro, you are probably not generating Match
  * errors where appropriate */
-#define LOOKUP_DRAWABLE(did, client)\
-    ((client->lastDrawableID == did) ? \
-     client->lastDrawable : (DrawablePtr)LookupDrawable(did, client))
+#define LOOKUP_DRAWABLE(did, client) (DrawablePtr)LookupDrawable(did, client)
 
 #ifdef XCSECURITY
 
 #define SECURITY_VERIFY_DRAWABLE(pDraw, did, client, mode)\
-    if (client->lastDrawableID == did && !client->trustLevel)\
-	pDraw = client->lastDrawable;\
-    else \
-    {\
-	pDraw = (DrawablePtr) SecurityLookupIDByClass(client, did, \
-						      RC_DRAWABLE, mode);\
-	if (!pDraw) \
-	{\
-	    client->errorValue = did; \
-	    return BadDrawable;\
-	}\
-	if (pDraw->type == UNDRAWABLE_WINDOW)\
-	    return BadMatch;\
-    }
+    pDraw = (DrawablePtr) SecurityLookupIDByClass(client, did, \
```

Comment:
You're adding a lot of whitespace changes in this file. What upstream did in this occasion was to leave indentation as it was (including the braces), so that the diff is smaller. Can you rework that in such a way? The benefit is leaving lines that didn't change as they were, so git blame will be better to understand. I'll give you an example for `SECURITY_VERIFY_DRAWABLE`, it would be great if you could apply the same logic to the other macros here.

Old code:
```C
#define SECURITY_VERIFY_DRAWABLE(pDraw, did, client, mode)\
    if (client->lastDrawableID == did && !client->trustLevel)\
	pDraw = client->lastDrawable;\
    else \
    {\
	pDraw = (DrawablePtr) SecurityLookupIDByClass(client, did, \
						      RC_DRAWABLE, mode);\
	if (!pDraw) \
	{\
	    client->errorValue = did; \
	    return BadDrawable;\
	}\
	if (pDraw->type == UNDRAWABLE_WINDOW)\
	    return BadMatch;\
    }
```

New code:
```C
#define SECURITY_VERIFY_DRAWABLE(pDraw, did, client, mode)\
    {\
	pDraw = (DrawablePtr) SecurityLookupIDByClass(client, did, \
						      RC_DRAWABLE, mode);\
	if (!pDraw) \
	{\
	    client->errorValue = did; \
	    return BadDrawable;\
	}\
	if (pDraw->type == UNDRAWABLE_WINDOW)\
	    return BadMatch;\
    }
```